customization-base/src/main/java/com/azure/autorest/customization/implementation/ls/EclipseLanguageServerFacade.java

```diff
@@ -40,6 +51,30 @@ public EclipseLanguageServerFacade(String pathToLanguageServerPlugin, int port)
         }
     }
 
+    private String unzipLanguageServer() throws IOException {
+        Path languageServerDirectory = Files.createTempDirectory("tmpjdt");
```

Comment:
Thoughts on instead of using a temporary directory we use a well-known directory that has different version identifiers based on the Autorest Java version being used? Similar to what is done for Test Proxy. This way we can remove the entire need to bundle the Eclipse language server into Autorest Java and can instead use the download links provided by Eclipse to download the ZIP file if it doesn't exist, if it does exist we know all this work has already been done and can be skipped.

Reply:
Yep, that will certainly reduce the size of the npm package. I have created an issue to track this https://github.com/Azure/autorest.java/issues/2227

Reply:
Great idea. We can just upload the zip to github release artifacts, and code the URL and SHA to code. Code can verify the local zip with SHA, if pass, use local, if not pass, download from github.